android/src/main/java/com/cloudwebrtc/webrtc/MethodCallHandlerImpl.java

```diff
@@ -1206,9 +1206,11 @@ public void mediaStreamAddTrack(final String streamId, final String trackId, Res
         }
       } else {
         resultError("mediaStreamAddTrack", "mediaStreamAddTrack() track [" + trackId + "] is null", result);
+        return;
       }
     } else {
       resultError("mediaStreamAddTrack", "mediaStreamAddTrack() stream [" + streamId + "] is null", result);
+      return;
     }
     result.success(null);
```

Comment:
I propose not to add `return`, but to move `result.success(null);` in the proper place. The flow will be much more readable.